server/api/crud/model_monitoring/deployment.py

```diff
@@ -384,14 +384,26 @@ def _initial_model_monitoring_stream_processing_function(
             server.api.api.utils.get_run_db_instance(self.db_session)
         )
 
+        secret_provider = server.api.crud.secrets.get_project_secret_provider(
+            project=self.project
+        )
+
+        tsdb_connector = mlrun.model_monitoring.get_tsdb_connector(
+            project=self.project, secret_provider=secret_provider
+        )
+        store_object = mlrun.model_monitoring.get_store_object(
+            project=self.project, secret_provider=secret_provider
```

Comment:
IIRC we have helper function for these in the server side

Reply:
So what's the suggestion here?